lib/installer.js

```diff
@@ -22,6 +23,12 @@ const WAD_EXE_MD5 = '50d694ebfaa622ef7e4061c1bf52efe6';
 class WADNotFoundError extends ES6Error {}
 
 const getWADExecutablePath = _.memoize(async function getWADInstallPath () {
+  if (WAD_INSTALLED_PATH) {
+    if (await fs.exists(WAD_INSTALLED_PATH)) {
+      return WAD_INSTALLED_PATH;
```

Comment:
it would be helpful to have a debug log here

Reply:
Added